app/modules/provisioning/users.py

```diff
@@ -31,3 +34,35 @@ def get_unique_users_from_groups(groups, key):
                 users_dict[str(user)] = user
     logger.info(f"Found {len(users_dict)} unique users.")
     return list(users_dict.values())
+
+
+def provision_users(
+    integration, operation, function, users, display_key=None, **kwargs
+):
```

Comment:
Likely just me missing it, but where is this new `provision_users` function being called from? Or are you planning to merge this in and then add the calls?

Reply:
Just making it available for now, not modifying the core feature yet! 😄